main.c

```diff
@@ -146,6 +123,35 @@ int main(int argc, char *argv[])
         }
     }
 
+    if(daemonize)
+        print_to_syslog = 1;
+
+    if(print_to_syslog)
+        openlog(APP_NAME, LOG_PID, LOG_DAEMON);
+
+    INFO("Version - " PACKAGE_VERSION "\n");
+
+    /* Sanity check */
```

Comment:
since you're moving this, could you make the sanity checks into a function and call it here ?
it would be nice to unclutter things a bit here

Reply:
Sure